<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropbox</string>
  </property>
  <widget class="QTreeWidget" name="ArbolArchivos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>371</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <column>
    <property name="text">
     <string>Tus Archivos</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="VerHistorialButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Historial de Modificaciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DescargarArchivoButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Descargar Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubirArchivoButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Archivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="EstadoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usuario Conectado: </string>
   </property>
  </widget>
  <widget class="QLabel" name="HistorialLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Historial de Modificaciones del Archivo: </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="HistorialTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>601</width>
     <height>221</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>122</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>33</number>
   </attribute>
   <column>
    <property name="text">
     <string>Fecha (dd:mm:aaaa)</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hora (hh:mm:ss)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tama√±o (bytes)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ActualizarButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CrearCarpetaButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Carpeta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NombreCarpetaLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>nombre_nueva_carpeta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DescargarCarpetaButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Descargar Carpeta</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="ArbolArchivos2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <column>
    <property name="text">
     <string>Tus Archivos</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="DestinoLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Destino Archivo A Mover:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>450</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MoverButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mover</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RenombrarButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Renombrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NuevoNombreLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>nuevo_nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AceptarButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelarButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>380</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
